src/components/database/Queries.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import {database} from '../../firebaseFeatures.js';
+import type {UserType, PendingType, ActionType} from '../admin/FlowTypes.js';
+
+function sanitize(collection: string,
```

Comment:
I would change this to use a `switch` statement.

There would be two ways to do this.
1. similar as before:
```js
function sanitize(collection, user) {
  switch (collection) {
    case 'active-members':
      return ({email: user.email, ......})
      break;
    .....
  }
}

// calling as:
sanitize(collection, user);
```
2. returning a function to manipulate the user:
```js
function sanitize(collection) {
  const sanitizeActiveMember = (user) => ({
     email: user.email,
     ....
  });

  switch (collection) {
     case 'active-members':
        return sanitzeActiveMember
     ....
  }
}

// calling as:
sanitize(collection)(user);
```